ovn/daemonset.sh

```diff
@@ -0,0 +1,335 @@
+master_loglevel=${MASTER_LOGLEVEL:-"4"}
+echo "master_loglevel: ${master_loglevel}"
+node_loglevel=${NODE_LOGLEVEL:-"4"}
+echo "node_loglevel: ${node_loglevel}"
+db_checker_loglevel=${DBCHECKER_LOGLEVEL:-"4"}
+echo "db_checker_loglevel: ${db_checker_loglevel}"
+ovn_loglevel_northd=${OVN_LOGLEVEL_NORTHD:-"-vconsole:info -vfile:info"}
+echo "ovn_loglevel_northd: ${ovn_loglevel_northd}"
+ovn_loglevel_nb=${OVN_LOGLEVEL_NB:-"-vconsole:info -vfile:info"}
+echo "ovn_loglevel_nb: ${ovn_loglevel_nb}"
+ovn_loglevel_sb=${OVN_LOGLEVEL_SB:-"-vconsole:info -vfile:info"}
+echo "ovn_loglevel_sb: ${ovn_loglevel_sb}"
+ovn_loglevel_controller=${OVN_LOGLEVEL_CONTROLLER:-"-vconsole:info"}
+echo "ovn_loglevel_controller: ${ovn_loglevel_controller}"
+ovn_loglevel_nbctld=${OVN_LOGLEVEL_NBCTLD:-"-vconsole:info"}
+echo "ovn_loglevel_nbctld: ${ovn_loglevel_nbctld}"
+ovnkube_logfile_maxsize=${OVNKUBE_LOGFILE_MAXSIZE:-"100"}
+echo "ovnkube_logfile_maxsize: ${ovnkube_logfile_maxsize}"
+ovnkube_logfile_maxbackups=${OVNKUBE_LOGFILE_MAXBACKUPS:-"5"}
+echo "ovnkube_logfile_maxbackups: ${ovnkube_logfile_maxbackups}"
+ovnkube_logfile_maxage=${OVNKUBE_LOGFILE_MAXAGE:-"5"}
+echo "ovnkube_logfile_maxage: ${ovnkube_logfile_maxage}"
+ovn_hybrid_overlay_enable=${OVN_HYBRID_OVERLAY_ENABLE}
+echo "ovn_hybrid_overlay_enable: ${ovn_hybrid_overlay_enable}"
+ovn_egress_ip_enable=${OVN_EGRESSIP_ENABLE}
+echo "ovn_egress_ip_enable: ${ovn_egress_ip_enable}"
+ovn_hybrid_overlay_net_cidr=${OVN_HYBRID_OVERLAY_NET_CIDR}
+echo "ovn_hybrid_overlay_net_cidr: ${ovn_hybrid_overlay_net_cidr}"
+ovn_disable_snat_multiple_gws=${OVN_DISABLE_SNAT_MULTIPLE_GWS}
+echo "ovn_disable_snat_multiple_gws: ${ovn_disable_snat_multiple_gws}"
+ovn_ssl_en=${OVN_SSL_ENABLE:-"no"}
+echo "ovn_ssl_enable: ${ovn_ssl_en}"
+ovn_unprivileged_mode=${OVN_UNPRIVILEGED_MODE:-"no"}
+echo "ovn_unprivileged_mode: ${ovn_unprivileged_mode}"
+ovn_nb_raft_election_timer=${OVN_NB_RAFT_ELECTION_TIMER:-1000}
+echo "ovn_nb_raft_election_timer: ${ovn_nb_raft_election_timer}"
+ovn_sb_raft_election_timer=${OVN_SB_RAFT_ELECTION_TIMER:-1000}
+echo "ovn_sb_raft_election_timer: ${ovn_sb_raft_election_timer}"
+ovn_master_count=${OVN_MASTER_COUNT:-"1"}
+echo "ovn_master_count: ${ovn_master_count}"
+ovn_remote_probe_interval=${OVN_REMOTE_PROBE_INTERVAL:-"100000"}
+echo "ovn_remote_probe_interval: ${ovn_remote_probe_interval}"
+ovn_nb_port=${OVN_NB_PORT:-6641}
+echo "ovn_nb_port: ${ovn_nb_port}"
+ovn_sb_port=${OVN_SB_PORT:-6642}
+echo "ovn_sb_port: ${ovn_sb_port}"
+ovn_nb_raft_port=${OVN_NB_RAFT_PORT:-6643}
+echo "ovn_nb_raft_port: ${ovn_nb_raft_port}"
+ovn_sb_raft_port=${OVN_SB_RAFT_PORT:-6644}
+echo "ovn_sb_raft_port: ${ovn_sb_raft_port}"
+ovn_multicast_enable=${OVN_MULTICAST_ENABLE}
+echo "ovn_multicast_enable: ${ovn_multicast_enable}"
+ovn_v4_join_subnet=${OVN_V4_JOIN_SUBNET}
+echo "ovn_v4_join_subnet: ${ovn_v4_join_subnet}"
+ovn_v6_join_subnet=${OVN_V6_JOIN_SUBNET}
+echo "ovn_v6_join_subnet: ${ovn_v6_join_subnet}"
+
+ovn_image=${image} \
```

Comment:
Wouldn’t it be simpler to export all the relevant variables?

Reply:
@skitt These files are taken mostly as-is from `ovn-kubernetes` Plan is to refactor all this to fit within our framework in subsequent PRs/commits. Wanted to get this initial PR up and ready so that @mangelajo and @sridhargaddam have something to work with. We need some changes in subm and operator before we can deploy subm on top of kind+ovn, so crucial to have something working asap.

Also, with this patch, we can enable a job that does `make clusters using=ovn` Would be good to have that in place before we start refactor.